src/content/blog/post-mortem-degraded-host-network.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Post-mortem: Degraded Host Network"
+generated: 1701894028874
+description: A single degraded host impacted certain Vals for some time.
+pubDate: Dec 31, 2024
+author: Max McDonnell
+---
+
+At 12:15am ET one of our servers started to experience long timeouts on both
+inbound and outbound connections. Existing connections continued to serve
+requests and some inbound and outbound requests were successful, but most
+requests would stall indefinitely in a pending state, timing out without
+establishing a connection. This persisted until 7:04am ET when we manually
+terminated the server after a user report.
```

Comment:
Should we mention the network degradation here?

Reply:
@zephraph updated, lmk what you think